src/scripts.js

```diff
@@ -23,6 +36,6 @@ const h20Data = new Hydration(h20Data);
 
 // An example of how you tell webpack to use a JS file
 
-// import userData from './data/users';
-// import UserRepository from './UserRepository';import Activity from './Activity';
-
+// import users from './data/users.js';
+// import UserRepository from './data/UserRepository.js';
+// import Activity from './Activity';
```

Comment:
This is the stuff that we messed with which broke something, right? We un-commented examples? Is it possible we just have to revert those changes and things will work again?

Reply:
@NEwertKrocker, it was those lines and then some of the `<script>` tags we were messing around with in our HTML file. 

Merge it up!